### ISSUE TITLE
Using removed Babel 5 option

### ISSUE THREAD

**Issue by tanmynguyen** [binggg/react-native-material-design-components#27]
Hi team!
When i apply mrn in my project react-native (v0.25), then I build....
![untitled](https://cloud.githubusercontent.com/assets/11315567/15081761/93277b08-13ee-11e6-9d10-62707ec9cc65.png)
Help me, please!!
Thanks!!!

**Comment by martinezguillaume** [comment]
same here

**Comment by happydog-intj** [comment]
+1

**Comment by nkaredia** [comment]
I'm getting same error. Build is successful but failing on device with this error.

![screenshot_2016-05-27-10-54-53](https://cloud.githubusercontent.com/assets/5784644/15611743/cf17ff96-23f9-11e6-80a3-ce64647b370c.png)

**Comment by rafaelcorreiapoli** [comment]
+1

**Comment by nevaryyy** [comment]
+1

**Comment by joncursi** [comment]
+1 on this TransformError:

![screen shot 2016-07-14 at 5 36 54 pm](https://cloud.githubusercontent.com/assets/4974609/16856691/8f974910-49e9-11e6-859d-1f331dccb15e.png)

**Comment by nevaryyy** [comment]
(Sorry, I'm not good at English)
I've solved this problem.

I think the reason is: 
           **Note: If your react-native version is 0.16+, For now can't support mrn**
My RN version is 0.28, so this problem occurred.

The solution is:
1. modify the file \node_modules\mrn.babelrc, delete "whitelist" and its value, add

```
"plugins": [
    "transform-regenerator",
    "syntax-flow"
    ],
  "presets": [
        "react", "es2015"
  ],
```

to this file. Finally like this;

**.babelrc**

```
{
  "retainLines": true,
  "compact": true,
  "comments": false,
  "sourceMaps": false,
  "plugins": [
    "transform-regenerator",
    "syntax-flow"
    ],
  "presets": [
        "react", "es2015"
  ],
}

```
1. Modify all the JS file in directory \node_modules\mrn\lib, except config, Icon and Index. 

**Before:**

```
import React, {
    Component,
    StyleSheet,
    PropTypes,
    View,
    Image
} from 'react-native';
```

**After:**

```
import React, {
    PropTypes,
    Component
} from 'react';
import {
    StyleSheet,
    View,
    Image
} from 'react-native';
```

In other words, import React and {PropTypes, Component} from 'react', import others from 'react-native'.

Not finished. Delete "export default" before "class _ViewName(e.g. Avatar)_“. And add "module.exports = _ViewName(e.g. Avatar)_;" to the end of the file.

The core of step 1 is babel, it changed a lot from babel 5 to babel6.
The core of step 2 is RN, if you have a high version RN, some features or syntax changed, so it can not support mrn.

Well, finally I run the project on device successful. But I think my solution is not good, hope someone provide a better one.

**Comment by RichardSleet** [comment]
It is still this issue !
